mautic/mautic
# Delete unused IP command - adding a limit, renaming the command

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Automated tests included? | N
| Related user documentation PR URL | N
| Related developer documentation PR URL | N
| Issues addressed (#s or URLs) | 
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
New command mautic:unusedip:delete that executes a delete query to remove duplicate IP addresses from the database. Added a limit parameter to restrict the number of deleted rows, default value is 10000.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Run mautic:unusedip:delete --limit 10000 command
3. IP Addresses from ip_addresses table without references in any other table should be deleted
